### PR TITLE
Remove qr code preprocessor directive

### PR DESCRIPTION
**Pull request by chrisfromwork**
Please view https://github.com/microsoft/MixedReality-SpectatorView/pull/308 before this pull request.

This review includes the following changes:
1) The preprocessor directive required to enable qr code detection is no longer needed. The QR Code binary is now included by default in the SpectatorView.Unity project.
2) A script is added to the repo that begins to enable nuget package generation for the spectator view source.
3) A SpectatorView.ExternalDependencies asmdef is added that references the QR Code binary.

Note: this change requires the following change in the MSBuildForUnity repo: https://github.com/microsoft/MSBuildForUnity/pull/94

### CODE REVIEW

File: tools/ci/scripts/createnugetpackages.ps1 [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    Where should we place the output? Defaults to ".\artifacts"
+.PARAMETER Version
+    What version of the artifacts should we build? If unspecified, the highest
+    git tag pointing to HEAD is searched. If none is found, an error is reported.
+#>
+param(
+    [string]$OutputDirectory = ".\artifacts",
+    [ValidatePattern("^\d+\.\d+\.\d+$")]
+    [string]$Version,
+    [string]$UnityDirectory
```
Comment (stephenatwork) [inline tools/ci/scripts/createnugetpackages.ps1]: .PARAMETER for this too?

Comment (chrisfromwork) [reply tools/ci/scripts/createnugetpackages.ps1]: sounds good.

File: tools/ci/scripts/createnugetpackages.ps1 [hunk 2]
```diff
@@ -0,0 +1,180 @@
+<#
+.SYNOPSIS
+    Builds the Mixed Reality Toolkit nuget packacges.
+.DESCRIPTION
+    Builds nuget packages for the Mixed Reality Toolkit.
```
Comment (fibann) [inline tools/ci/scripts/createnugetpackages.ps1]: s/Mixed Reality Toolkit/SpectatorView ?

File: tools/ci/scripts/createnugetpackages.ps1 [hunk 3]
```diff
@@ -0,0 +1,180 @@
+<#
```
Comment (fibann) [inline tools/ci/scripts/createnugetpackages.ps1]: This file seems unrelated to QR codes, and also maybe unfinished (the actual nuget creation seems commented out). Should this be moved to a different PR?

Comment (chrisfromwork) [reply tools/ci/scripts/createnugetpackages.ps1]: id like to keep this file, it lets us validate the QR Code references are resolving correctly

Comment (fibann) [reply tools/ci/scripts/createnugetpackages.ps1]: Sounds reasonable, maybe just add a TODO explaining the commented part if we plan to make it work eventually.

File: src/SpectatorView.Unity/Assets/PhotoCapture/Microsoft.MixedReality.PhotoCapture.msb4u.csproj.meta [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+ScriptedImporter:
+  fileIDToRecycleName:
+    11400000: Microsoft.MixedReality.PhotoCapture.msb4u
+  externalObjects: {}
+  userData: 
+  assetBundleName: 
+  assetBundleVariant: 
+  script: {fileID: 11500000, guid: 0e4dca1acba66fd478b3854f09a32ec7, type: 3}
+  buildEngine: 0
+  profiles: []
```
Comment (matthejo) [inline src/SpectatorView.Unity/Assets/PhotoCapture/Microsoft.MixedReality.PhotoCapture.msb4u.csproj.meta]: Should these new meta files exist? Seems like something that might be missing from the .gitignore since I don't see the corresponding files, just the meta files.

Comment (chrisfromwork) [reply src/SpectatorView.Unity/Assets/PhotoCapture/Microsoft.MixedReality.PhotoCapture.msb4u.csproj.meta]: I'm not sure what the call is on this. I'll ask andrei today.

Comment (chrisfromwork) [reply src/SpectatorView.Unity/Assets/PhotoCapture/Microsoft.MixedReality.PhotoCapture.msb4u.csproj.meta]: we want to include these files, there may be something wrong if we aren't including csprojs .I will follow up in a different review.
